codalab/objects/work_manager.py

```diff
@@ -175,6 +175,25 @@ def check_bundle_actions(self):
 
         return did_something
 
+    def check_timed_out_bundles(self, update_timeout=60*60*24, creation_timeout=60*60*24*30):
```

Comment:
You don't need the creation_timeout...as long as a job is running and getting updated, that means there's a worker out there who you can send a kill to (and we have another mechanism for timeouts).  We just need the update_timeout, because that's where there's no worker to talk to anymore (presumably).